app/src/main/java/com/iyehuda/feelslike/data/model/Post.kt

```diff
@@ -0,0 +1,11 @@
+package com.iyehuda.feelslike.data.model
+
+data class Post(
+    val id: String = "",
+    val username: String = "",
```

Comment:
It should be the author user id (since username is not unique and may be changed)